packages/crawler/src/index.ts

```diff
@@ -55,8 +62,9 @@ interface Arguments {
         simulate: args.simulate,
         selectors: args.selectors,
         localOutputDir: args.output,
+        maxRequestsPerCrawl: args.maxUrls,
     });
 })().catch((error) => {
-    console.log('Exception: ', System.serializeError(error));
+    console.log('Exception: ', error);
```

Comment:
did serialization has an issue?

Reply:
yeah it cause an error while running the crawler

`TypeError: Reflect.hasOwnMetadata is not a function
    at C:\Users\ahmohame.REDMOND\Documents\GitHub\accessibility-insights-service\node_modules\inversify\lib\annotation\injectable.js:7:21
    at __decorate (C:\Users\ahmohame.REDMOND\Documents\GitHub\accessibility-insights-service\packages\common\dist\ciphers\hash-generator.js:5:95)     at Object.<anonymous> (C:\Users\ahmohame.REDMOND\Documents\GitHub\accessibility-insights-service\packages\common\dist\ciphers\hash-generator.js:55:17)
    at Generator.next (<anonymous>)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
error Command failed with exit code 1.`

![image](https://user-images.githubusercontent.com/45858632/90839749-171bcf80-e30d-11ea-8033-0eb91e633d89.png)

Reply:
add to the index.ts as first import: `import 'reflect-metadata';`